### DISCUSSION TITLE
How to change Middle Mouse Button to Left Mouse Button for pausing?

### DISCUSSION THREAD

**Discussion by Xiarno** [mpvnet-player/mpv.net#686] [Q&A] [answered]
I am trying to switch the Middle Mouse Button to Left Mouse Button to pause the video player. However, it doesn't work at all for some reason.

I've used the input configurator to try to do that without success.
![15123_mpvnet_JsAzTEAO2y](https://github.com/mpv-player/mpv/assets/24660485/43ea5947-3014-486f-8cab-b83694291fa5)

There's this thing that is there for some reason. Could it be the reason that it isn't working at all? I am unable to unassign this "ignore". Each time I try to remove it, it comes back and I don't understand why.
![15124_mpvnet_Zpb38x9Ra6](https://github.com/mpv-player/mpv/assets/24660485/bb234713-84f8-4d9d-b22c-be44c4cf4b38)

How can I do it please? I wanna keep all the other option for pausing too (Like spacebar).
And no I do not care about the issue of dragging with left click anywhere on the screen (That's mostly the thing I've found when searching for a solution on this problem :/)

Thanks!

**Comment by Sneakpeakcss** [comment, marked as answer]
If multiple commands use the same key then only the very last is actually active, so normally that `ignore` shouldn't matter at all. 

The issue here is that when you tried to edit `MBTN_MID` to `MBTN_LEFT` then every binding with the same name (`Play/Pause`) was overwritten, and `mpv.net` doesn't treat it as new binding even if it actually created `input.conf` and added it there. 

If you also edited the `Name` field to anything else then it would actually work properly… It's just some editor jankyness / issue with parsing commands with the exact same comment as `mpv.net` built-in defaults.

At this point you either reset to default by removing `input.conf` in `C:\Users\%username%\AppData\Roaming\mpv.net`, and then again use the editor while also changing `Name` field, or you add your bindings manually to `input.conf`, just to avoid any other potential issues, which is the best option really, because every binding in there takes priority over `mpv.net` defaults and you don't have to worry that the editor will mess it up even more:


```
MBTN_LEFT   script-message-to mpvnet play-pause
MBTN_MID    ignore
```

**Reply by Xiarno** [reply]
I did actually try that but it seems the ignore just comes back no matter what and it just doesn't want to accept the left mouse button at all.

**Reply by Sneakpeakcss** [reply]
> I did actually try that but it seems the ignore just comes back no matter what and it just doesn't want to accept the left mouse button at all.

It will always come back because it's the built-in default, if you correctly added it through editor, or `input.conf` then it should have another `MBTN_LEFT` that's below it, taking priority over the `ignore` binding:

![image](https://github.com/mpvnet-player/mpv.net/assets/77424331/8a2dfdbf-1e39-472f-96de-ddeae7b1e2f9)

**Reply by Xiarno** [reply]
> > I did actually try that but it seems the ignore just comes back no matter what and it just doesn't want to accept the left mouse button at all.
> 
> It will always come back because it's the built-in default, if you correctly added it through editor, or `input.conf` then it should have another `MBTN_LEFT` that's below it, taking priority over the `ignore` binding:
> 
> [image]

Ok so I tried that again just to be sure and somehow it worked this time? Idk why because I remember thinking the same thing as you and trying it and it not working. I guess I must've done something different after all. Oh well. Thanks for the answer & explanation!
